### ISSUE TITLE
HoverOffsetYScale on AppBar doesn't do anything

### ISSUE THREAD

**Issue by BenWoodford** [microsoft/MixedRealityToolkit-Unity#2962]
**Does this affect the legacy HoloToolkit (master) or the Mixed Reality Toolkit (mrtk_release)?**
HoloToolkit 

**Describe the bug**
It's not referenced anywhere in the code it seems, at least not in 2017.4.2.0. Changing it has no effect on the AppBar

**To Reproduce**
Change the HoverOffsetYScale on the AppBar - the example should work for this I assume.

**Expected behavior**
Not sure what is meant to happen with it but I assume the Y coordinate on the bar should change to an extent?

**Actual behavior**
Nothing

**Comment by Yoyozilla** [comment]
Closing older HTK issue as we are no longer actively working on HTK.

Please upgrade to MRTK V2 for the latest fixes.

**Comment by FlorianJa** [comment]
In MRTK RC2.1 the bug is still there. Please check https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/AppBar/AppBar.cs
You will see HoverOffsetYScale  is not used in the code.

**Comment by kwmeyer** [comment]
I just ran into this issue myself, and can confirm that it is still present in MRTK RC2.1. `HoverOffsetYScale` is exposed as a property, but is not actually used anywhere in `AppBar.cs` to determine the position of the AppBar. It appears that this was introduced during the rework done in #2212.

Please consider reopening this issue.

**Comment by kwmeyer** [comment]
I couldn't resist playing around with this a bit, and I found a solution that at least works for me. Not sure how it worked in HTK before, but here's what I did. in `AppBar.cs`, add this after the line where the final position is calculated:

`finalPosition += (helper.GetFaceCentroid(followingFaceIndex) - helper.GetFaceBottomCentroid(followingFaceIndex)).normalized * hoverOffsetYScale;`

Basically finds the upwards direction along the following face, scales it by the aforementioned property, then adds it to the app bar's final position.
